recipes/itak/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2.0.2" %}
+{% set name = "itak" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kentnf/iTAK/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e7209cfbb131f77fee2a5e998813f79a979016d1d7ba77a6cc078d220c9a576a
+
+build:
+  number: 0
+  noarch: python
+  run_exports:
+    weak:
+      - {{ pin_subpackage('itak', max_pin="2.0.2") }}
```

Comment:
the max_ping does not look correct. Please have a look at our recommendation in the PR description.

Reply:
> the max_ping does not look correct. Please have a look at our recommendation in the PR description.

Thank you for suggestion, the max_pin has been fixed, please let me know if there is any other problem with it.